services/api-server/src/simcore_service_api_server/api/routes/studies.py

```diff
@@ -88,14 +88,25 @@ async def get_study(
 
 
 @router.post(
-    "/{study_id:uuid}",
+    "/{study_id:uuid}:clone",
     response_model=Study,
+    status_code=status.HTTP_201_CREATED,
     responses={**_COMMON_ERROR_RESPONSES},
     include_in_schema=API_SERVER_DEV_FEATURES_ENABLED,
 )
-async def clone_study(study_id: StudyID):
-    msg = f"cloning study with study_id={study_id!r}. SEE https://github.com/ITISFoundation/osparc-simcore/issues/4651"
-    raise NotImplementedError(msg)
+async def clone_study(
+    study_id: StudyID,
```

Comment:
good point. I did not think about it.
if you clone a template with this entrypoint, it will produce another template (and not a standard study)
For your tests, i recommend you to create a standard study in one of the puppetteer and clone in from there